package.json

```diff
@@ -148,6 +148,7 @@
 		"grapheme-splitter": "^1.0.2",
 		"gridfs-stream": "^1.1.1",
 		"he": "^1.1.1",
+		"highcharts": "^6.2.0",
```

Comment:
did you added `highcharts` for some reason? :sweat_smile:

Reply:
lol, I'm not sure why that happened, but it is gone already haha